mapswipe_workers/mapswipe_workers/project_types/base/project.py

```diff
@@ -217,7 +223,10 @@ def save_to_firebase(self, project, groups, groupsOfTasks):
                     json.dumps(tasks_list).replace(" ", "").replace("\n", "")
                 )
                 compressed_tasks = gzip_str.gzip_str(json_string_tasks)
-                encoded_tasks = base64.b64encode(compressed_tasks)
+                # we need to decode back, but only when using Python 3.6
+                # when using Python 3.7 it just works
+                # Unfortunately the docker image uses Python 3.7
```

Comment:
I'm confused by this comment. It sounds like you're sad that it just works?? is there a typo somewhere?

Reply:
right! The docker image uses python 3.6